components/uuf-renderablecreator-hbs/src/test/java/org/wso2/carbon/uuf/handlebars/HbsHelperTest.java

```diff
@@ -255,7 +258,8 @@ public void testPublicWithMultipleParameters() {
     @Test
     public void testSecured() {
         String content = "{{#secured}} secured content {{else}} un-secured content {{/secured}}";
-        String output = createRenderable(content).render(null, createLookup(), createRequestLookup(), createAPI());
+        String output = createRenderable(content).render(null, createLookup(), createRequestLookup(),
```

Comment:
Seems like this line doesn't exceed 120 limit. Hence, revert this formatting change.